Makefile

```diff
@@ -40,8 +39,16 @@ run:
 env:
 	./scripts/bootstrap-virtualenv.sh
 
-lint:
-	${PYTHONLINT} ${PYDIRS}
+lint: listoutdated flake8diff
+
+listoutdated:
+	pip list --outdated --allow-external=cafe,cloudcafe
+
+flake8diff:
+	git diff --patch --no-prefix origin/master | flake8 --diff
```

Comment:
That's definitely an improvement. I had something similar at first, but then realized that I wanted the current staging area linted as well. I suppose it's better to just do it in two steps instead :)